drivers/ukintctlr/gic/Makefile.uk

```diff
@@ -0,0 +1,14 @@
+$(eval $(call addplatlib_s,kvm,libukintctlr_gic,$(CONFIG_LIBUKINTCTLR_GIC)))
```

Comment:
After rebasing to #1023 this no longer needs to register as a platform library, and use `addlib_s` instead:
```
$(eval $(call addlib_s,libukintctlr_gic,$(CONFIG_LIBUKINTCTLR_GIC)))
```